tests/units/anta_tests/test_services.py

```diff
@@ -0,0 +1,106 @@
+            "result": "failure",
+            "messages": ["`mlagasu`: Not found."],
+        },
+    },
+    {
+        "name": "failure-reason-disabled",
+        "test": VerifyErrdisableRecovery,
+        "eos_data": [
+            """
+                        Errdisable Reason              Timer Status   Timer Interval
+                        ------------------------------ ----------------- --------------
+                        acl                            Disabled                 300
+                        arp-inspection                 Enabled                  300
+                        bgp-session-tracking           Enabled                  30
+                    """
+        ],
+        "inputs": {"reasons": [{"reason": "acl", "interval": 300}, {"reason": "bgp-session-tracking", "interval": 30}]},
+        "expected": {
+            "result": "failure",
+            "messages": ["`acl`:\nExpected `Enabled` as the status, but found `Disabled` instead."],
+        },
+    },
+    {
+        "name": "failure-interval-not-ok",
+        "test": VerifyErrdisableRecovery,
+        "eos_data": [
+            """
+                        Errdisable Reason              Timer Status   Timer Interval
+                        ------------------------------ ----------------- --------------
+                        acl                            Enabled                  300
+                        arp-inspection                 Enabled                  300
+                        bgp-session-tracking           Enabled                  30
+                    """
+        ],
+        "inputs": {"reasons": [{"reason": "acl", "interval": 30}, {"reason": "bgp-session-tracking", "interval": 30}]},
+        "expected": {
+            "result": "failure",
+            "messages": ["`acl`:\nExpected `30` as the interval, but found `300` instead."],
+        },
+    },
+    {
+        "name": "failure-all-type",
+        "test": VerifyErrdisableRecovery,
+        "eos_data": [
+            """
+                        Errdisable Reason              Timer Status   Timer Interval
+                        ------------------------------ ----------------- --------------
+                        acl                            Disabled                 300
+                        arp-inspection                 Enabled                  300
+                        bgp-session-tracking           Enabled                  30
+                    """
+        ],
+        "inputs": {"reasons": [{"reason": "acl", "interval": 30}, {"reason": "bgp-session-tracking", "interval": 300}, {"reason": "mlagasu", "interval": 30}]},
+        "expected": {
+            "result": "failure",
+            "messages": [
+                "`acl`:\nExpected `30` as the interval, but found `300` instead.\nExpected `Enabled` as the status, but found `Disabled` instead.",
+                "`bgp-session-tracking`:\nExpected `300` as the interval, but found `30` instead.",
```

Comment:
Not sure if the `\n` makes sens in these error messages, especially whnen it is not in all of them (mlagasu)

Reply:
Added \n when reason not found. The though for adding \n is give better readability of failure message in table format. 
Output with \n
```
                                                                                    All tests results                                                                                     
┏━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━┓
┃ Device    ┃ Test Name                ┃ Test Status ┃ Message(s)                                              ┃ Test description                                         ┃ Test category ┃
┡━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━┩
│ s1-spine1 │ VerifyErrdisableRecovery │ failure     │ `acl`:                                                  │ Verifies the error-disable recovery reason, status, and  │ services      │
│           │                          │             │ Expected `30` as the interval, but found `300` instead. │ interval.                                                │               │
│           │                          │             │ Expected `Enabled` as the status, but found `Disabled`  │                                                          │               │
│           │                          │             │ instead.                                                │                                                          │               │
│           │                          │             │                                                         │                                                          │               │
│           │                          │             │ `tapagg`:                                               │                                                          │               │
│           │                          │             │ Expected `30` as the interval, but found `300` instead. │                                                          │               │
│           │                          │             │ Expected `Enabled` as the status, but found `Disabled`  │                                                          │               │
│           │                          │             │ instead.                                                │                                                          │               │
│           │                          │             │                                                         │                                                          │               │
└───────────┴──────────────────────────┴─────────────┴─────────────────────────────────────────────────────────┴──────────────────────────────────────────────────────────┴───────────────
```

Output without \n
```
                                                                                     All tests results                                                                                     
┏━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━┓
┃ Device    ┃ Test Name                ┃ Test Status ┃ Message(s)                                              ┃ Test description                                         ┃ Test category ┃
┡━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━┩
│ s1-spine1 │ VerifyErrdisableRecovery │ failure     │ `acl`:Expected `30` as the interval, but found `300`    │ Verifies the error-disable recovery reason, status, and  │ services      │
│           │                          │             │ instead.Expected `Enabled` as the status, but found     │ interval.                                                │               │
│           │                          │             │ `Disabled` instead.                                     │                                                          │               │
│           │                          │             │ `tapagg`:Expected `30` as the interval, but found `300` │                                                          │               │
│           │                          │             │ instead.Expected `Enabled` as the status, but found     │                                                          │               │
│           │                          │             │ `Disabled` instead.                                     │                                                          │               │
└───────────┴──────────────────────────┴─────────────┴─────────────────────────────────────────────────────────┴──────────────────────────────────────────────────────────┴───────────────┘
```

Let me know if its look ok